bugalink-backend/trips/tests.py

```diff
@@ -59,4 +57,59 @@ def test_get_trip_by_search(self):
         "prefers_talk=False&allows_pets=False&allows_smoking=False"
         response = self.client.get(url)
         data = json.loads(response.content)
-        self.assertEqual(data["trips"][0]["id"], self.trip_2.id)
+        self.assertEqual(data[0]["id"], self.trip_2.id)
+
+
+class TripRateTest(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+        load_complex_data(self)
+        self.client.force_authenticate(user=self.user_2)
+
+    def test_rate_trip(self):
+        url = "/api/v1/trips/" + str(self.trip.pk) + "/rate/"
+
+        self.client.post(
+            url,
+            data={
+                "rating": 2.3,
+                "is_good_driver": True,
+                "is_pleasant_driver": False,
+                "already_knew": True,
+            },
+        )
+
+        self.assertEqual(
+            DriverRating.objects.get(trip_request=self.trip_request).rating, 2.3
+        )
+
+class RequestTrip(TestCase):
```

Comment:
These are tests "pa salir del paso" as they don't test any unitary functionality, if-else branches... such that the method could not work as expected and still return a code 201. In fact, there aren't any tests that make sure that invalid data results in errors and the objects not being created in the DB, which are probably the most important tests (everyone tests the happy path but there are always edge cases and misuse in the app)